docs/deploying-identity-provider.md

```diff
@@ -0,0 +1,73 @@
+# Deploying an Identity Provider for Fulcio Integration
```

Comment:
Pretty simple question (I think I already know the answer too), this is within a private sigstore context, not the PGS?